kuflow-temporal-activity-s3/src/test/java/com/kuflow/temporal/activity/s3/S3ActivitiesTest.java

```diff
@@ -82,7 +82,7 @@ public void givenAProcessItemWhenCopyDocumentToS3ThenDocumentIsUploaded() {
         processItemTask.setData(data);
 
         ProcessItem processItem = new ProcessItem();
-        processItem.setId(UUIDUtils.generateUUIDv7());
+        processItem.setId(UUID.randomUUID());
```

Comment:
Although it is still being tested, it may be better to use generate KuFlowWorkflow.generateUUIDv7.

But it's not a workflow, so I'd leave the generation utility generic.

Reply:
I needed a UUID v7 generator library to do that because Java doesn't have this support now.

So I can create the utility using a uuid v7 library and add the dependency optionally or use the uuid v7 library only on tests.

Reply:
What do you think?

Reply:
But then what is this **KuFlowWorkflow.generateUUIDv7()**

Reply:
Is to allow create uuid in v7 version, but uses classes that come from temporal, see https://github.com/kuflow/kuflow-sdk-java/pull/96/files#diff-b3cb46285eb75908f41b26675a988b28f1d2c1895b49aae42e549aa55f9b9adeR25

So we avoid to add any external library and is aligned with how the other SDKs generate a uuid v7